### PR TITLE
Use the proper unit

### PR DESCRIPTION
**Pull request by xtophe80**
Brief summary of the changes
----------------------------

Fix typo in radio frequency unit.

### CODE REVIEW

File: src/InfoBoxes/Panel/RadioEdit.cpp [hunk 1]
```diff
@@ -30,8 +30,7 @@ Copyright_License {
 
 class RadioOffsetButtons final : public OffsetButtonsWidget {
 public:
-  RadioOffsetButtons(bool active_freq) noexcept
-    :OffsetButtonsWidget(UIGlobals::GetDialogLook().button, _T("%.0f KHz"), 5, 1000),set_active_freq(active_freq) {}
+  RadioOffsetButtons(bool active_freq):OffsetButtonsWidget(UIGlobals::GetDialogLook().button, _T("%.0f kHz"), 5, 1000),set_active_freq(active_freq) {}
```
Comment (lordfolken) [inline src/InfoBoxes/Panel/RadioEdit.cpp:33]: * Readd the noexcept
* Readd the linebreak

Make your commit message more meaningful:
InfoBox/RadioEdit: fix typo in frequency unit

Comment (lordfolken) [reply src/InfoBoxes/Panel/RadioEdit.cpp:33]: Thank you for contributing!

**Review by lordfolken — CHANGES_REQUESTED** [review]

**Comment by MaxKellermann** [comment]
The commitdiff is unnecessarily confusing; it changes the coding style, removes `noexcept`, when all it should really do is change 'K' to 'k'.
The commit message is not helpful.

**Comment by MaxKellermann** [comment]
b19dd6e4234aa78a16c9f6978d8df0da2d2c1762

**Pull request closed without merging**